docs/agents/vantage_pro2.rst

```diff
@@ -0,0 +1,80 @@
+.. highlight:: rst
+
+.. _vantagepro2:
+
+=============
+Vantage Pro2 Agent
+=============
+
+The Davis Instruments Vantage Pro2 is a weather system + monitor used to 
+acquire and readout weather data. The Vantage Pro2 monitor is connected to
+the labratory computer via usb cable and data is sent though that connection.
+
+.. argparse::
+    :filename: ../agents/vantagePro2_agent/vantage_pro2_agent.py
+    :func: make_parser
+    :prog: python3 vantage_pro2_agent.py
+
+Description
+-----------
+Out of the box, one just needs to connect the weather station to the 
+Vantage Pro2 monitor, and the monitor to the lab computer via usb. As long as
+the Vantage Pro2 monitor is on, the intitialize and data acquisition tasks 
+should work "out of the box", as it where.
+
+The Vantage Pro2 monitor and weather station records 
+
+Configuration File Examples
+---------------------------
+Below are configuration examples for the ocs config file and for running the
+Agent in a docker container.
+
+ocs-config
+``````````
+To configure the Vantage Pro2 Agent we need to add a VantagePro2Agent block to our ocs
+configuration file. Here is an example configuration block using all of the
+available arguments::
+
+     {'agent-class': 'VantagePro2Agent',
+       'instance-id': 'vantagepro2agent',
+       'arguments': [['--port', '/dev/ttyUSB0'],
```

Comment:
This `/dev/ttyUSB0` port will change depending on how many other usb devices are plugged in. There may be udev rules that assigned a fixed device port based on the serial number. See the LS240 agent as an example: https://github.com/simonsobs/socs/blob/015f25e9ad9b93b610971bfe93d6a1c4252df83e/agents/lakeshore240/LS240_agent.py#L339-L345

I'd suggest looking on the system you're testing with and figure out a fixed way to determine the device port based on the hardware info.